packages/webpack-cli/lib/groups/runHelp.js

```diff
@@ -90,46 +98,85 @@ const outputHelp = (args) => {
             logger.raw(flags);
         }
     } else {
-        const negatedFlags = flags
-            .filter((flag) => flag.negative)
-            .reduce((allFlags, flag) => {
-                return [...allFlags, { name: `no-${flag.name}`, description: `Negates ${flag.name}`, type: Boolean }];
-            }, []);
+        let flagsToDisplay = flags.filter(({ help }) => help === 'base'); // basic options only one word
+
+        const negatedFlags = (allFlags) => {
+            return allFlags
+                .filter((flag) => flag.negative)
+                .reduce((allFlags, flag) => {
+                    return [...allFlags, { name: `no-${flag.name}`, description: `Negates ${flag.name}`, type: Boolean }];
+                }, []);
+        };
+
         const title = bold('⬡                     ') + underline('webpack') + bold('                     ⬡');
         const desc = 'The build tool for modern web applications';
         const websitelink = '         ' + underline('https://webpack.js.org');
         const usage = bold('Usage') + ': ' + '`' + green('webpack [...options] | <command>') + '`';
         const examples = bold('Example') + ': ' + '`' + green('webpack help --flag | <command>') + '`';
         const hh = `          ${title}\n\n${websitelink}\n\n${desc}\n\n${usage}\n${examples}`;
-        const output = commandLineUsage([
-            { content: hh, raw: true },
-            {
-                header: 'Available Commands',
-                content: commands.map((cmd) => {
-                    return { name: `${cmd.name} | ${cmd.alias}`, summary: cmd.description };
-                }),
-            },
-            {
-                header: 'Options',
-                optionList: flags
-                    .map((e) => {
-                        if (e.type.length > 1) {
-                            e.type = e.type[0];
-                        }
-
-                        // Here we replace special characters with chalk's escape
-                        // syntax (`\$&`) to avoid chalk trying to re-process our input.
-                        // This is needed because chalk supports a form of `{var}`
-                        // interpolation.
-                        e.description = e.description.replace(/[{}\\]/g, '\\$&');
-
-                        return e;
-                    })
-                    .concat(negatedFlags),
-            },
-        ]);
-
-        logger.raw(output);
+
+        if (args.includes('verbose') || args.includes('--help=verbose')) {
```

Comment:
Very bad - `args.includes('verbose')`

Reply:
`help` doesn't get parsed like other flags. This check here is to support `webpack --help verbose`

Reply:
We really should refactor this hack stuff, `commander` provide great ability to do it https://github.com/tj/commander.js?#custom-help, no need to look at `args`, it is bad, I will merge this PR, but the next is refactor, no PR to help while we don't migrate on `commander` API

Reply:
makes sense 👍🏻